pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSources.java

```diff
@@ -133,6 +134,12 @@ void runCmd() throws Exception {
         @Parameter(names = "--sourceConfigFile", description = "The path to a YAML config file specifying the "
                 + "source's configuration")
         protected String sourceConfigFile;
+        @Parameter(names = "--cpu", description = "The cpu that need to be allocated per function instance(applicable only to docker runtime)")
```

Comment:
probably want to specify the units for each of the resources in the description for clarity